### PR TITLE
Update for activeadmin 1

### PR DESCRIPTION
**Pull request by brianswko**
Addresses breaking changes introduced in https://github.com/activeadmin/activeadmin/pull/3486

### CODE REVIEW

**Comment by brianswko** [comment]
@11mdlow take a gander? planning to bump the major version after this is merged in.

File: lib/ActiveAdmin-Globalize3-inputs.rb [hunk 1]
```diff
@@ -1,3 +1,2 @@
 require 'formtastic/globalize_inputs'
-require 'active_admin/globalize_inputs'
```
Comment (11mdlow) [inline lib/ActiveAdmin-Globalize3-inputs.rb:2]: ah, so is globalize inputs included in this version of AA?

Comment (brianswko) [inline lib/ActiveAdmin-Globalize3-inputs.rb:2]: it's not, but if you look at the AA PR linked above they've refactored how the forms are built. This file had previously been included only to wrap around the formtastic file but is no longer necessary.

Comment (11mdlow) [inline lib/ActiveAdmin-Globalize3-inputs.rb:2]: Ah I see...sorry totally skimmed over the PR description :-P

**Comment by 11mdlow** [comment]
:ship:

**Pull request merged**